src/components/common/Stepper.vue

```diff
@@ -1,17 +1,12 @@
 <template>
-  <v-container id="step-buttons-container">
```

Comment:
Please revert your changes to this file.

Reply:
poor rebase on my part, corrected in latest commit